tests/e2e/test_pv_scale_ocs_create_delete_pvcs.py

```diff
@@ -0,0 +1,338 @@
+        # Lists to evaluate the pod and pvc created during function call
+        temp_pod_objs, temp_pvc_objs = ([] for i in range(2))
+        future_cephfs_rwo_obj, future_cephfs_rwx_obj = ([] for i in range(2))
+        future_rbd_rwo_obj, future_rbd_rwx_obj = ([] for i in range(2))
+        future_pod_objs, future_temp_pod_objs = ([] for i in range(2))
+        # Flatten to remove the sublist created during future obj
+        flatten = lambda l: [item for sublist in l for item in sublist]
+        self.delete_pod_count = round(number_of_pvc / 2)
+        log.info(f"Inside create_pvc_pod function")
+        # Using Thread to create multiple PVC in parallel
+        # Below code creates pvc with CephFS RWX & RWO and RBD RWX & RWO
+        with ThreadPoolExecutor() as executor:
+            for mode in access_modes:
+                if mode == constants.ACCESS_MODE_RWO:
+                    future_cephfs_rwo_obj.append(
+                        executor.submit(helpers.create_multiple_pvcs,
+                                        sc_name=cephfs_sc_obj.name, namespace=self.namespace,
+                                        number_of_pvc=number_of_pvc, access_mode=mode,
+                                        size=size))
+                else:
+                    future_cephfs_rwx_obj.append(
+                        executor.submit(helpers.create_multiple_pvcs,
+                                        sc_name=cephfs_sc_obj.name, namespace=self.namespace,
+                                        number_of_pvc=number_of_pvc, access_mode=mode,
+                                        size=size))
+        cephfs_rwo_pvcs = [pvc_obj.result() for pvc_obj in future_cephfs_rwo_obj]
+        cephfs_rwo_pvcs = flatten(cephfs_rwo_pvcs)
+        cephfs_rwx_pvcs = [pvc_obj.result() for pvc_obj in future_cephfs_rwx_obj]
+        cephfs_rwx_pvcs = flatten(cephfs_rwx_pvcs)
+        temp_pvc_objs.extend(cephfs_rwo_pvcs)
+        temp_pvc_objs.extend(cephfs_rwx_pvcs)
+        with ThreadPoolExecutor() as executor:
+            for mode in access_modes:
+                if mode == constants.ACCESS_MODE_RWO:
+                    future_rbd_rwo_obj.append(
+                        executor.submit(helpers.create_multiple_pvcs,
+                                        sc_name=cephfs_sc_obj.name, namespace=self.namespace,
+                                        number_of_pvc=number_of_pvc, access_mode=mode,
+                                        size=size))
+                else:
+                    future_rbd_rwx_obj.append(
+                        executor.submit(helpers.create_multiple_pvcs,
+                                        sc_name=rbd_sc_obj.name, namespace=self.namespace,
+                                        number_of_pvc=number_of_pvc, access_mode=mode,
+                                        size=size))
+        rbd_rwo_pvcs = [pvc_obj.result() for pvc_obj in future_rbd_rwo_obj]
```

Comment:
same

Reply:
soon will provide a patch using functions, which will remove the identical code.

Reply:
done